filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
```

Comment:
`a` would be included in `*args`, and this function doesn't seem to do anything particular to `a` except append it to `*args`.

This could just be:

    def only_one_true(*args)